skopt/learning/forest.py

```diff
@@ -3,6 +3,44 @@
 from sklearn.ensemble import ExtraTreesRegressor as sk_ExtraTreesRegressor
 
 
+def _return_std(X, trees, predictions):
+    """
+    Return standard deviation in predictions given a list of trees.
```

Comment:
I was rather saying that we are computing the standard deviation of the target `y`, rather than the standard deviation of the prediction `\hat(y)` of the target. For me "standard deviation in predictions"  meant the later.